test/inductor/test_cpu_cpp_wrapper.py

```diff
@@ -42,6 +47,19 @@
 )
 
 
+def is_windows() -> bool:
```

Comment:
Why not directly using `IS_WINDOWS` later?

Reply:
@desertfire I use function and mark `TODO` comments.